MetaMorpheus/EngineLayer/CrosslinkSearch/CrosslinkSearchEngine.cs

```diff
@@ -320,9 +320,10 @@ private void FindCrosslinkedPeptide(Ms2ScanWithSpecificMass scan, List<(double,
                 possibleMatches = new List<CrosslinkSpectralMatch>();
             }
 
+            var initiatorMethionine = ((DigestionParams)CommonParameters.DigestionParams).InitiatorMethionineBehavior;
             foreach (var id in idsOfPeptidesPossiblyObserved)
```

Comment:
Should there be some kind of null check here? What happens if CommonParameters.DigestionParams is an RnaDigestionParams object

Reply:
I am assuming that we will not use the crosslink search engine for RNA without serious refactoring that would catch this assumption.

Reply:
Would it require a serious refactor? Why can't you just use a
`if (CommonParameters.DigestionParams is not DigestionParams digestionParams)`
check and throw an exception if false?

Reply:
> Why can't you just use a `if (CommonParameters.DigestionParams is not DigestionParams digestionParams)` check and throw an exception if false?

That is the affect of the current implementation. I can add more explicit checks if you'd like.